lib/Driver/ToolChains.cpp

```diff
@@ -119,6 +119,22 @@ static void addCommonFrontendArgs(const ToolChain &TC,
     arguments.push_back(inputArgs.MakeArgString(OI.SDKPath));
   }
 
+  // HACK: Add an -iquote Clang search path (using -Xcc) in order to cope with
+  // differences between precompiled and non-precompiled bridging headers.
+  //
+  // In non-PCH contexts, a module map in the same directory as the bridging
+  // header is read in by the Clang importer. To replicate that behavior with
+  // PCH, add an extra -iquote search path at the start of the search list.
+  // This won't affect any textual header resolution because the directory of
+  // the current file is already searched for quoted includes.
```

Comment:
One thought I had was that you could look for a module map file directory and use `-fmodule-map-file <path>` if you found one.  The downside is that *forces* parsing the module map file you found, which is also a behaviour change vs. looking it up on demand.  I don't have a good intuition for which is less disruptive.